policyengine_canada/variables/gov/provinces/qc/tax/income/credits/cce/cce_eligible_nondisabled_old_children copy.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class cce_eligible_nondisabled_old_children(Variable):
+    value_type = int
+    entity = Household
+    label = "Quebec childcare tax credit eligible nondisabled old children"
+    definition_period = YEAR
+
+    adds = ["cce_eligible_nondiabled_old_child"]
```

Comment:
Could you please add an integration test to catch these?
```suggestion
    adds = ["cce_eligible_nondisabled_old_child"]
```